saleor/dashboard/discount/views.py

```diff
@@ -124,7 +124,7 @@ def voucher_delete(request, pk):
         instance.delete()
         messages.success(
             request,
-            pgettext_lazy('Voucher message', 'Removed voucher %s') % (instance,))
+            pgettext_lazy('Voucher message', 'Removed voucher %s') % instance)
```

Comment:
Please always explicitly pass either a tuple or a dict when interpolating strings with `%`.

Reply:
Ok, I'll restore it.